docs/reference/settings/ilm-settings.asciidoc

```diff
@@ -37,8 +37,11 @@ You can explicitly set it to <<skipping-rollover, skip rollover>>.
 Defaults to `false`.
 
 `index.lifecycle.name`::
-(<<indices-update-settings, Dynamic>>, string) 
-The name of the policy to use to manage the index.
+(<<indices-update-settings, Dynamic>>, string)
+The name of the policy to use to manage the index. As the current executing phase
+definition is cached in the index metadata, changing the setting to a different policy
+will not affect the currently executing phase, but the next phase will be read from
+the newly configured policy.
```

Comment:
This seems to say the same thing as https://www.elastic.co/guide/en/elasticsearch/reference/master/update-lifecycle-policy.html#ilm-apply-new-policy. I wonder if we should just add an xref to there?

Reply:
@jrodewig great point! I missed that we document this already. An xref sounds great

Reply:
Awesome. I pushed 28f7fce15e4a59ab8c9a69939b731929503a89eb to add an xref.